src/app/board/page.js

```diff
@@ -1,5 +1,27 @@
+import Board from "@/components/board/Board";
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div className="flex justify-center items-center p-3">
+      <div className="grid grid-cols-1">
+        <Title text="Get To Know Us" />
```

Comment:
<img width="1385" alt="image" src="https://github.com/acm-ucr/h4b-website/assets/91030482/fc7f61dd-75cc-47e9-ac42-f1117f9dc783">

add spacing between the title and cards.

i would also pull changes from dev and changing the colors (based on the Figma file)

Reply:
about the changing colors, for the title the tailwind config file does not have the lighter green shown in the figma file. Should I add it as like green-300?

Reply:
yes 👍